juju/client/facade.py

```diff
@@ -926,11 +926,22 @@ def generate_definitions(schemas):
     return definitions
 
 
-def generate_facades(schemas):
+def sortable_schema_version(version_string: str) -> Tuple[int, int, int]:
+    """Return a sorting key in the form (major, minor, micro) from a version string."""
+    # 'latest' is special cased in load_schemas and should come last
+    if version_string == 'latest':
+        return (9000, 9000, 9000)
+    # raise ValueError if string isn't in the format A.B.C
+    major, minor, micro = version_string.split('.')
```

Comment:
wdyt about using what already do for the client version:

```
from packaging import version
version.parse(server_version)
```

juju/client/connector.py

Reply:
Sounds like a good idea, if you want to take care of it to push this over the finish line that would be cool, otherwise I'll hopefully get to it some time during the sprint